index.js

```diff
@@ -235,7 +236,23 @@ const legacyScriptingRunner = (Zap, zobj, app) => {
       } else {
         funcs.push(response => {
           response.throwForStatus();
-          return zobj.JSON.parse(response.content);
+          const data = zobj.JSON.parse(response.content);
+          if (!ensureArray) {
```

Comment:
I suppose this is what we'll use for the connection label call?

Reply:
Look for where we call `runEventCombo` with `ensureArray` being `false`. Currently, only `post_oauthv2_token` scripting method use this.

Reply:
Sorry I was confused. Currently, this is used for OAuth2 to get and refresh access token.